build_scripts/release_scripts/create_rondb_tarball.sh

```diff
@@ -7,21 +7,61 @@ OUTPUT_DIR_ABS=$3
 
 cd $TEMP_BUILD_DIR_ABS
 
+# The following code of how to determine OS and CPU architecture is not optimal.
+# Essentially, we are trying to keep the naming conventions on repo.hops.works
+#   Examples:
+#   rondb-21.04.2-linux-glibc2.17-x86_64.tar.gz
+#   rondb-21.04.2-macos-xcode-13.1-arm64_v8.tar.gz
```

Comment:
xcode? it is an IDE

Reply:
Xcode comes with all the C/C++ libraries in Mac. It's more than an IDE.